e2e/dotcms-e2e-node/frontend/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "@pages/*": ["./src/pages/*"],
+      "@pages": ["./src/pages/index"],
```

Comment:
@nicobytes Do you have a README, with what is in each folder (Like architecture) and the pattern? It will help to provide context to the generation of the E2E test.